### PR TITLE
Fix the updated ackee-tracker API

### LINKED ISSUE #7: Update to ackee-tracker 5 breaks svelte-ackee
Just a heads-up:

The tracking API has changed with ackee-tracker version 5, and breaks `svelte-ackee` after the ackee-tracker dependency was bumped [in a recent commit](https://github.com/gaia-green-tech/svelte-ackee/commit/3db6f8cab6faff1b2ecee5158b93d572e280cdea).

Previously the tracker was created with a server object and then options.
The new API takes the server domain when creating the tracker, but the domain ID as part of recording a page visit.

See the [example here](https://github.com/electerious/ackee-tracker#programmatic):

```js
import * as ackeeTracker from 'ackee-tracker'

ackeeTracker.create('https://ackee.example.com').record('hd11f820-68a1-11e6-8047-79c0c2d9bce0')
```

### PR DESCRIPTION
**Pull request by suda**
Fixes #7 

@sebastian could you perhaps confirm that this fixes the issue?

### CODE REVIEW

**Comment by sebastian** [comment]
TL;DR: the version of the svelte-ackee tracker as per this pull-request doesn't build on my end at all.

Caveat: I am using SvelteKit (currently version 1.0.0-next.109) rather than Sapper. This might impact the results, or in fact be the reason it fails for me.

The way you use the ackee-tracker looks right to me. I had made a similar change.
The reason I didn't make a pull-request was that the I cannot get the newer ackee-tracker versions to compile with SvelteKit.

I am seeing the following.

```
ReferenceError: navigator is not defined
    at Object.S [as create] (/<path>/node_modules/ackee-tracker/dist/ackee-tracker.min.js:1:18935)
    at Proxy.useAckeeSapper (/<path>/node_modules/svelte-ackee/src/ackee.js:37:38)
    at __layout.svelte:10:37
    at Object.$$render (/<path>/node_modules/svelte/internal/index.js:1380:22)
    at root.svelte:36:44
    at $$render (/<path>/node_modules/svelte/internal/index.js:1380:22)
    at Object.render (/<path>/node_modules/svelte/internal/index.js:1388:26)
    at render_response (file:////<path>/node_modules/@sveltejs/kit/dist/ssr.js:377:28)
    at async respond_with_error (file:////<path>/node_modules/@sveltejs/kit/dist/ssr.js:936:10)
    at async respond$1 (file:////<path>/node_modules/@sveltejs/kit/dist/ssr.js:1155:10)
```

The only way I could get things to work was a super ugly hack whereby I:
- bundle a version of ackee-tracker with my project
- pass `window` along in the `create` call when instantiating the Ackee tracker, and then rewrite all instances of `navigator` and `location` to `window.navigator` and `window.location`... 

This neither seem like a proper nor a clean solution...

**Comment by suda** [comment]
Hmmm if it's rendered in the SSR context the `navigator` object wouldn't be present. So there should be a way to turn off Ackee for the SSR (it wouldn't work anyway). I haven't dig into SvelteKit yet, but do you mind raising this issue there as well? I wonder if there's a "correct" way of disabling features just for the SSR.

**Comment by suda** [comment]
@sebastian I think I know what's the issue. Are you running it in the `load` function? Because:

> Code called inside load blocks:
> ...
> should not reference window, document, or any browser-specific objects

So that would crash because of `navigator` not being accessible there. I think tracking code shouldn't be called during SSR anyway so the Ackee initialization should be in the `onMount` callback. I will merge it as it is and if we find a "correct" way to initialize it in SvelteKit (we also moved our code to it), we'll add clear information about it (and example) to this repo :)
